src/components/ClusterTemplateQuotas/NewQuotaDialog/AccessFields.tsx

```diff
@@ -20,14 +20,14 @@ export const useUsers = () => {
   return useK8sWatchResource<User[]>({
     groupVersionKind: userGVK,
     isList: true,
-  });
+  }) as CorrectWatchK8sResult<User[]>;
```

Comment:
instead of using `as` everywhere, lets just create a wrapper for `useK8sWatchResource` which will return correct types

Reply:
Done